frontend/src/components/Inquiries/InquiriesTable.tsx

```diff
@@ -56,49 +68,131 @@ const InquiriesTable = () => {
     })
   }, [inquiries])
 
+  const queryClient = useQueryClient()
```

Comment:
All of this code is confusing.  InquiriesTable component should only have code for the Inquiries Table.  Can't the Edit function be imported from another file?  And isn't  InquiryForm the component that you would import?

Reply:
refactoring/optimizations will be done in different pr. I already combined Edit and Create Inquiry into one per your request. :)

Reply:
Yeah.  You did a good job with that.  Thanks.  
The issue here is that I think that component should be used here.
Inquiries Table should not know about the queryClient().  
I thought the whole point of refactoring was to reuse that logic for the InquiryForm.  
I have to look more carefully at your code  to tell what's going on.

Reply:
I agree with Mitchell. We shouldn't be merging code we know needs to be refactored immediately. It's not just about functional correctness -- the code has to conform to standards, and this isn't quite it.